front-end/src/helpers/validateAllRequired.js

```diff
@@ -1,13 +1,42 @@
+import moment from 'moment';
```

Comment:
I've created only one validateAllRequired.js file for all forms, but we would need a separate file foe each form, as I see it. Unless you'll find a way to share some validations between the forms.
I suggest to store each file in the matching form folder, rather than in 'helpers'.